pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
 addopts = --verbose --showlocals --cov=girder --cov-append --cov-config=.coveragerc --cov-report=""
+cache_dir = build/test/cache/pytest
```

Comment:
@danlamanna Is there a reason this needs to be nested, and can't be `pytest_cache`? Are there other test caches we're planning on supporting?